.github/workflows/connect-npm-deploy.yml

```diff
@@ -0,0 +1,66 @@
+name: "[Deploy] connect npm deploy"
+on:
+  workflow_call:
+    inputs:
+      package:
+        description: Name of the package to deploy.
+        type: choice
+        options:
+          - blockchain-link-types
+          - blockchain-link-utils
+          - blockchain-link
+          - connect-common
+          - transport
+          - utils
+          - utxo-lib
+          - connect-plugin-stellar
+          - connect-plugin-ethereum
+          - analytics
+          - connect-analytics
+          - type-utils
+          - env-utils
+          - protocol
+          - protobuf
+          - schema-utils`
+
+jobs:
+  deploy-npm-beta:
+    name: node
+    runs-on: ubuntu-latest
+    # Only run on branches like: `npm-release/**`
+    # TODO: we probably should add here some environment
```

Comment:
@vdovhanych are those environment ready or we do not need them yet?

Reply:
we have these environments so far:
- develop
- staging
- production

we could add also lets say npm-packages or something like that so again we can limit where that npm token is available

Reply:
Creating specific env for npm makes sense to me.

Reply:
So are we sticking with develop staging production for now? 
And add npm environment later?

Reply:
I think it would be better to add it now since the beginning.
So I updated as you suggested to `environment: npm-packages`. Let me know if you think differently.